app/services/push_notifications/post_broadcaster_service.rb

```diff
@@ -0,0 +1,19 @@
+module PushNotifications
+  class UsersBroadcasterService
```

Comment:
you're still working on it, right? because I don't see this class used.

Reply:
this class will be used once we implement the logic behind sending push notifications to users :)